subsys/lorawan/lorawan.c

```diff
@@ -495,6 +495,54 @@ int lorawan_set_datarate(enum lorawan_datarate dr)
 	return 0;
 }
 
+int lorawan_set_channel_mask(enum lorawan_channel_mask mask, bool set_default)
+{
+	MibRequestConfirm_t mib_req;
+
+	uint64_t pri_mask = 0;
+	uint32_t sec_mask = 0;
+	uint16_t channel_mask[6] = { 0 };
```

Comment:
Does the 6 need to match with REGION_NVM_CHANNELS_MASK_SIZE or are they unrelated

Reply:
Not related.